### PR TITLE
ydcv-rs: update to 0.4.7

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

ydcv-rs: update to 0.4.7

- Enable LTO
- Use Source tarball
- Use SRCS

Package(s) Affected
-------------------

ydcv-rs 0.4.7

Security Update?
----------------

- [ ] Yes
- [x] No

Architectural Progress
----------------------

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-i18n/ydcv-rs/autobuild/defines [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PKGNAME=ydcv-rs
 PKGDES="A Rust version of ydcv"
 PKGDEP="openssl dbus libxcb"
-BUILDDEP="git cargo python-3"
+BUILDDEP="git cargo python-3 llvm"
```
Comment (MingcongBai) [inline extra-i18n/ydcv-rs/autobuild/defines]: Please switch `cargo` to `rustc`, as `cargo` is now a transitional package which will (eventually) be removed.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-i18n/ydcv-rs/autobuild/defines (hunk 1 above)
Comment (eatradish) [reply extra-i18n/ydcv-rs/autobuild/defines]: Done

**Review by MingcongBai — APPROVED** [review]

**Comment by eatradish** [comment]
Loongson3 FTBFS: http://fars.ee/H9sG

**Comment by MingcongBai** [comment]
Post-Merge Architectural Progress
----------------------



- [x] AMD64 `amd64`
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
--------------------------------



Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

**Pull request merged**